bucket/strokesplus.net.json

```
@@ -0,0 +1,39 @@
+{
+    "version": "0.5.6.6",
+    "description": "Mouse gesture recognition utility for Windows which allows you to create powerful mouse gestures that save you time.",
+    "homepage": "https://www.strokesplus.net",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.strokesplus.net/FAQ"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://cdn.strokesplus.net/files/StrokesPlus.net_Portable_0.5.6.6.zip",
+            "hash": "b1c82604910676bc7bb3cd43b02a278bc01f51b212a6c180aab59f0a6ec86e9f"
+        },
+        "32bit": {
+            "url": "https://cdn.strokesplus.net/files/StrokesPlus.net_Portable_0.5.6.6.zip",
+            "hash": "b1c82604910676bc7bb3cd43b02a278bc01f51b212a6c180aab59f0a6ec86e9f"
+        }
+    },
+    "shortcuts": [
+        [
+            "StrokesPlus.net.exe",
+            "StrokesPlus.net"
+        ]
+    ],

```

```suggestion
    "pre_install": "if (!(Test-Path \"$persist_dir\\strokesplussettings.json\")) { Copy-Item \"$dir\\Default_StrokesPlus.net.json\" \"$dir\\strokesplussettings.json\" | Out-Null }",
    "shortcuts": [
        [
            "StrokesPlus.net.exe",
            "StrokesPlus.net",
            "settingsfile=\"$dir\\strokesplussettings.json\""
        ]
    ],
    "persist": "strokesplussettings.json",
    "pre_uninstall": [
        "# The following must be done as StrokesPlus deletes the original file which was linked to the persist folder when it saves settings, rather than editing the file.",
        "Copy-Item \"$dir\\strokesplussettings.json\" \"$persist_dir\\strokesplussettings.json\" | Out-Null"
    ],
```

Allows settings to be persisted. If it's a first time install, the default settings are copied, so no error on startup.

---

ok. I'm confused about `strokesplussettings.json` I have checked and am sure the correct file is `StrokePlus.net.json` and can confirm that `Default_StrokePlus.net.json` exists
When I change the setting of my strokeplus to a custom setting, `Default_StrokePlus.net.json` disappeared and `StrokePlus.net.json` appeared.

---

It looks like this doesn't account for the other file `StrokesPlus.net.json` when it only copies `Default_Strokesplus.net.json`.  
Is there a way to add `Strokesplus.net.json`??

---

No need to account for `StrokesPlus.net.json` as it is only created when no custom setting file is provided. Scoop's shortcut makes the app use the custom one `strokesplussettings.json`.

---

alright. I understand now. 
👍

---

```suggestion
    "shortcuts": [
        [
            "StrokesPlus.net.exe",
            "StrokesPlus.net",
            "settingsfile=\"$dir\\strokesplussettings.json\""
        ]
    ],
```
While updating to 0.5.6.7, you missed out this line and so it stopped loading the custom settings file, please add it back and then I can approve this PR.